docs/UserGuide.md

```diff
@@ -61,9 +61,29 @@ Welcome HR executives and Department Heads! Coydir is a desktop app to efficient
 
 ## Using this Guide
 
+
 ### Navigating the User Guide
+This document might look daunting at first! But fear not, in this section, you will learn how to use 
+this user guide like a professional.
+
+####Interested in how to setup? 
+
+You can head over to [Quick Start](#quick-start) to install and run Coydir.
+
+####Interested in what Coydir can do?
+
+You can head over to [Features](#features) to learn how to use the features 
+offered in Coydir.
+
+Here are some symbols to look out for while using this guide:
```

Comment:
This part, I think, should move up in front, because it applies for the entire document and not just the "Features" part.

Reply:
Sorry what do you mean ah?